packages/suite/src/components/guide/GuideSearch.tsx

```diff
@@ -42,9 +42,13 @@ const PreviewContent = styled.div`
 const StyledInput = styled(Input)`
     && {
         background-color: ${({ theme }) => theme.BG_GREY_ALT};
-        border: none;
         border-radius: 8px;
         height: 40px;
+        border-color: ${props => props.theme.BG_GREY_ALT};
+
+        :focus {
+            border-color: ${({ theme }) => theme.STROKE_GREY};
```

Comment:
Invisible in darkmode

Reply:
536346ac0d0d6adcdd9e7ed0120ca6df31dbaa18